utils/consul-metrics-logger

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
This is a strayed file that got into the latter update to the patch. Sorry. I have removed it from https://github.com/Seagate/cortx-hare/pull/1404, kindly update.
Thank you.

Reply:
Oh, Ok. Don't you think it should be kept?

Reply:
Ah, sorry - it's an empty file! :) Will remove it also.